File: nivel-00/antecessor-e-sucessor-de-um-numero-qualquer/foo.c
#include <stdio.h>
#include <assert.h>

int antecessor(int numero) {
    return numero - 1;
}

int sucessor(int numero) {
    return numero + 1;
}

int main (){

    //
    // Testes
    //
    assert(9 == antecessor(10));
    assert(11 == sucessor(10));

    return 0;
}

File: nivel-00/antecessor-e-sucessor-de-um-numero-qualquer/foo.js
var assert = require('assert');

var antecessor = function(numero) {
    return numero - 1;
}

var sucessor = function(numero) {
    return numero + 1;
}


//
// Seu teste
//
try {
    assert.equal(9, antecessor(10), "deve retornar o antecessor");
    assert.equal(11, sucessor(10), "deve retornar o sucessor");
} catch(e) {
    console.log(e);
}